modules/hal_nordic/nrfs/Kconfig

```diff
@@ -0,0 +1,121 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+config NRF_HAS_NRFS
+	bool
+
+config NRFS_HAS_CLOCK_SERVICE
+	bool
+
+config NRFS_HAS_CONST_LATENCY_SERVICE
+	bool
+
+config NRFS_HAS_DIAG_SERVICE
+	bool
+
+config NRFS_HAS_DVFS_SERVICE
+	bool
+
+config NRFS_HAS_MRAM_SERVICE
+	bool
+
+config NRFS_HAS_PMIC_SERVICE
+	bool
+
+config NRFS_HAS_RESET_SERVICE
+	bool
+
+config NRFS_HAS_TEMP_SERVICE
+	bool
+
+config NRFS_HAS_VBUS_DETECTOR_SERVICE
+	bool
+
+menu "nRF Services"
+	depends on NRF_HAS_NRFS
+
+config NRFS
+	bool "nRF Services Support"
+	help
+	  This option enables the nRF Services library
+
+config NRFS_LOCAL_DOMAIN
+	bool "nRF Services Local Domain Support"
+	depends on NRFS
+	help
+	  This option enables the nRF Services Local Domain libraries
+
+config NRFS_BACKEND_IPC_SERVICE
+	bool "IPC service backend for NRFS"
+	depends on NRFS
+	select IPC_SERVICE
+	select IPC_SERVICE_BACKEND_ICMSG
+	select MBOX
+	select EVENTS
+	select REBOOT
+	help
+	  This option enables transport protocol for the NRFS
```

Comment:
How does it enables the transport protocol? I cannot find any use of `CONFIG_NRFS_BACKEND_IPC_SERVICE`.

Reply:
fixed